File: src/main/java/me/itzisonn_/meazy/runtime/interpreter/InvalidIdentifierException.java
package me.itzisonn_.meazy.runtime.interpreter;

import me.itzisonn_.meazy.lang.TextException;
import me.itzisonn_.meazy.lang.text.Text;

/**
 * Is thrown when {@link EvaluationFunction} can't find object with requested identifier
 */
public class InvalidIdentifierException extends TextException {
    /**
     * @param text Text
     */
    public InvalidIdentifierException(Text text) {
        super(text);
    }
}

File: src/main/java/me/itzisonn_/meazy/runtime/InvalidIdentifierException.kt
package me.itzisonn_.meazy.runtime;

import me.itzisonn_.meazy.text.TextException;
import me.itzisonn_.meazy.text.Text;
import org.jspecify.annotations.NullMarked;

/**
 * Is thrown when resolver can't find object with requested identifier
 */
@NullMarked
public class InvalidIdentifierException extends TextException {
    /**
     * @param text Text
     */
    public InvalidIdentifierException(Text text) {
        super(text);
    }
}
